### PR TITLE
Adding Fantom Deployed Addresses

### PR DESCRIPTION
**Pull request by mjdatadog**
Added deployment addresses for ftmUSD pool.

### CODE REVIEW

File: contracts.md [hunk 1]
```diff
@@ -194,3 +194,17 @@ All Saddle protocol smart contracts are immutable.
...
+| `AmplificationUtils`      | [0x2b7A5a5923ECa5C00c6572Cf3E8e08384F563F93](https://ftmscan.com/address/0x2b7a5a5923eca5c00c6572cf3e8e08384f563f93#code) |
+| `LPToken`                 | [0xc0409EC303b727Bc1F511d7F8C71FD5Ead96De1c](https://ftmscan.com/address/0xc0409EC303b727Bc1F511d7F8C71FD5Ead96De1c#code) |
+| `SwapFlashLoan`           | [0xBea9F78090bDB9e662d8CB301A00ad09A5b756e9](https://ftmscan.com/address/0xBea9F78090bDB9e662d8CB301A00ad09A5b756e9#code) |
+| `SwapUtils`               | [0x4f6A43Ad7cba042606dECaCA730d4CE0A57ac62e](https://ftmscan.com/address/0x4f6A43Ad7cba042606dECaCA730d4CE0A57ac62e#code) |
+| `SaddleFtmUSDPool`        | [0xBea9F78090bDB9e662d8CB301A00ad09A5b756e9](https://ftmscan.com/address/0xBea9F78090bDB9e662d8CB301A00ad09A5b756e9#code) |
+| `SaddleOptUSDPoolLPToken` | [0xc969dd0a7ab0f8a0c5a69c0839db39b6c928bc08](https://ftmscan.com/address/0xc969dd0a7ab0f8a0c5a69c0839db39b6c928bc08#code) |
+| `SDL`                     | [0xa29b548056c3fd0f68bad9d4829ec4e66f22f796](https://ftmscan.com/address/0x2b7a5a5923eca5c00c6572cf3e8e08384f563f93#code) |
```
Comment (ug02fast) [inline contracts.md:210]:
```suggestion
| `SDL`                     | [0xa29b548056c3fd0f68bad9d4829ec4e66f22f796](https://ftmscan.com/address/0xa29b548056c3fd0f68bad9d4829ec4e66f22f796#code) |
```

these are diff

**Review by ug02fast — CHANGES_REQUESTED** [review]

File: contracts.md (hunk 1 above)
Comment (hammeiam) [reply contracts.md:210]: I don't think SDL is deployed on FTM at al
Screen Shot 2022-03-25 at 1 50 50 PM
l

File: contracts.md [hunk 2]
```diff
@@ -194,3 +194,17 @@ All Saddle protocol smart contracts are immutable.
...
+| :------------------------ | :------------------------------------------------------------------------------------------------------------------------------------ |
+| `AmplificationUtils`      | [0x2b7A5a5923ECa5C00c6572Cf3E8e08384F563F93](https://ftmscan.com/address/0x2b7a5a5923eca5c00c6572cf3e8e08384f563f93#code) |
+| `LPToken`                 | [0xc0409EC303b727Bc1F511d7F8C71FD5Ead96De1c](https://ftmscan.com/address/0xc0409EC303b727Bc1F511d7F8C71FD5Ead96De1c#code) |
+| `SwapFlashLoan`           | [0xBea9F78090bDB9e662d8CB301A00ad09A5b756e9](https://ftmscan.com/address/0xBea9F78090bDB9e662d8CB301A00ad09A5b756e9#code) |
+| `SwapUtils`               | [0x4f6A43Ad7cba042606dECaCA730d4CE0A57ac62e](https://ftmscan.com/address/0x4f6A43Ad7cba042606dECaCA730d4CE0A57ac62e#code) |
+| `SaddleFtmUSDPool`        | [0xBea9F78090bDB9e662d8CB301A00ad09A5b756e9](https://ftmscan.com/address/0xBea9F78090bDB9e662d8CB301A00ad09A5b756e9#code) |
+| `SaddleOptUSDPoolLPToken` | [0xc969dd0a7ab0f8a0c5a69c0839db39b6c928bc08](https://ftmscan.com/address/0xc969dd0a7ab0f8a0c5a69c0839db39b6c928bc08#code) |
```
Comment (hammeiam) [inline contracts.md:209]: this should be `SaddleFtmUSDPoolLPToken`

File: contracts.md (hunk 1 above)
Comment (penandlim) [reply contracts.md:210]: I believe SDL is not bridged to FTM yet

**Pull request merged**